metaflow/plugins/aip/aip_cli.py

```diff
@@ -260,6 +260,13 @@ def cli_decorator(func: Callable):
             "If not set, the default iam role associated with the pod will be used",
             show_default=True,
         )
+        @click.option(
+            "--add-default-cards",
+            "add_default_cards",
+            default=True,
+            help="Whether to add default card to all workflow steps",
+            show_default=True,
+        )
```

Comment:
Taleb: Allow reading this from env var using metaflow config, so that it's more CICD friendly.